.github/ISSUE_TEMPLATE/submission_request.md

```diff
@@ -6,28 +6,27 @@ labels: ''
 assignees: ''
 ---
 
-**Note**: Please read [this article](https://docs.genicsblog.com/author/create-a-submission-request) before continuing further.
+<!-- **Note**: Please read [this article](https://docs.genicsblog.com/author/create-a-submission-request) before continuing further. -->
```

Comment:
We can remove the markdown in the comment block and add normal text referring to the link. Example: `<!-- Please read this document before continuing further: https://docs.genicsblog.com/author/create-a-submission-request -->`

Reply:
Done